features/step_definitions/see_dishes_steps.rb

```diff
@@ -0,0 +1,19 @@
+When(/^I visit the dishes page$/) do
+  visit '/dishes'
+end
+
+Then(/^I should see the page$/) do
+  save_and_open_page
+end
+
+Then(/^I see "([^"]*)"$/) do |arg1|
```

Comment:
Replace `arg1` with something like `content` or `text`. Try not to leave `arg1` in your step defs - replace it with a descriptive variable name so your colleagues can understand your tests.